Tests/MultiSetsTests/CountedSet/CountedSet+SetAlgebra Tests.swift

```diff
@@ -0,0 +1,159 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Collections open source project
+//
+// Copyright (c) 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import MultiSets
+
+import _CollectionsTestSupport
+
+
+private let x: CountedSet<Character> = ["a": 1, "b": 2, "c": 3, "d": 4]
+private let y: CountedSet<Character> = ["e", "f", "a", "f"]
```

Comment:
Instead of hardwiring these particular values, these tests should be using the combinatorial testing features provided in the `_CollectionsTestSupport` package to exhaustively exercise the implementations over many different combinations of inputs, generated automatically.

Reply:
I was pretty sure I something like that existed, but decided to get the tests hammered out for an initial pull request before looking into it.